src/pages/Xvs/Header/index.tsx

```diff
@@ -1,13 +1,107 @@
 /** @jsxImportSource @emotion/react */
-import React from 'react';
-import { Paper } from '@mui/material';
+import React, { useMemo } from 'react';
+import BigNumber from 'bignumber.js';
+import { Paper, Typography } from '@mui/material';
+import { EllipseText, Icon, LabeledProgressBar } from 'components';
+import copy from 'copy-to-clipboard';
+import { getToken, generateBscScanUrl } from 'utilities';
+import { formatCoinsToReadableValue } from 'utilities/common';
+import { useTranslation } from 'translation';
+import { useStyles } from '../styles';
+import { MINTED_XVS_WEI } from '../constants';
 
 interface IHeaderProps {
   className?: string;
 }
 
-const HeaderUi: React.FC<IHeaderProps> = ({ className }) => <Paper className={className} />;
+interface IHeaderContainerProps {
+  remainingDistribution: BigNumber;
+  dailyVenus: BigNumber;
+  venusVaiVaultRate: BigNumber;
+  totalXvsDistributedWei: BigNumber;
+}
+
+export const HeaderUi: React.FC<IHeaderProps & IHeaderContainerProps> = ({
+  className,
+  remainingDistribution,
+  dailyVenus,
+  venusVaiVaultRate,
+  totalXvsDistributedWei,
+}) => {
+  const styles = useStyles();
+  const xvsAddress = getToken('xvs').address;
+  const copyAddress = () => copy(xvsAddress);
+  const { t } = useTranslation();
+
+  const readableDailyDistribution = useMemo(() => {
+    const dailyDistribution = dailyVenus.div(new BigNumber(10).pow(18)).plus(venusVaiVaultRate);
```

Comment:
Let's use `getToken` to get the token decimals rather than inputting `18` directly